roles/lib_openshift/src/ansible/oc_user.py

```diff
@@ -0,0 +1,34 @@
+# pylint: skip-file
+# flake8: noqa
+
+def main():
+    '''
+    ansible oc module for user
+    '''
+
+    module = AnsibleModule(
+        argument_spec=dict(
+            kubeconfig=dict(default='/etc/origin/master/admin.kubeconfig', type='str'),
+            state=dict(default='present', type='str',
+                       choices=['present', 'absent', 'list']),
+            debug=dict(default=False, type='bool'),
+            username=dict(default=None, type='str'),
+            full_name=dict(default=None, type='str'),
+            # setting groups for user data will not populate the
+            # 'groups' field in the user data.
+            # it will call out to the group data and make the user
+            # entry there
```

Comment:
This info is good to know, but seems like it should be in the module docs and not on the argument_spec here.